website/content/docs/concepts/seal.mdx

```diff
@@ -157,255 +142,247 @@ and their API equivalents in the [/sys/init](/vault/api-docs/system/init) endpoi
   the API endpoint.
 - `recovery-pgp-keys`: The PGP keys to use to encrypt the returned recovery
   key shares. This value is equivalent to the `recovery_pgp_keys` value in the
-  API endpoint, although as with `pgp_keys` the object in the API endpoint is
+  API endpoint, although as with `pgp_keys`. the object in the API endpoint is
   an array, not a string.
 
-Additionally, Vault will refuse to initialize if the option has not been set to
-generate a key, and no key is found. See
+Additionally, Vault does not initialize if you have not set the option to
+generate a key, and Vault does not find a key. Refer to
```

Comment:
Removes future tense, passive voice. Changes "see" to "refer to" for better accessibility.